packages/jsii-pacmak/test/expected.jsii-calc-base-of-base/python/src/scope/jsii_calc_base_of_base/__init__.py

```diff
@@ -2,13 +2,13 @@
 import builtins
 import datetime
 import enum
+import publication
 import typing
 
 import jsii
 import jsii.compat
-import publication
 
-__jsii_assembly__ = jsii.JSIIAssembly.load("@scope/jsii-calc-base-of-base", "1.1.0", __name__, "jsii-calc-base-of-base@1.1.0.jsii.tgz")
+__jsii_assembly__ = jsii.JSIIAssembly.load("@scope/jsii-calc-base-of-base", "1.1.0", "scope.jsii_calc_base_of_base", "jsii-calc-base-of-base@1.1.0.jsii.tgz")
```

Comment:
Will this break the world, or is `__name__` equivalent?

Reply:
`__name__` is equivalent in the existing places, and is not (as needed -- which is why it changed) in submodules.